examples/consistency_models/train_consistency_distillation.py

```diff
@@ -0,0 +1,770 @@
+    logger.info(accelerator.state, main_process_only=False)
+    if accelerator.is_local_main_process:
+        datasets.utils.logging.set_verbosity_warning()
+        diffusers.utils.logging.set_verbosity_info()
+    else:
+        datasets.utils.logging.set_verbosity_error()
+        diffusers.utils.logging.set_verbosity_error()
+
+    # Handle the repository creation
+    if accelerator.is_main_process:
+        if args.push_to_hub:
+            if args.hub_model_id is None:
+                repo_name = get_full_repo_name(Path(args.output_dir).name, token=args.hub_token)
+            else:
+                repo_name = args.hub_model_id
+            repo_id = create_repo(repo_name, exist_ok=True, token=args.hub_token).repo_id
+            repo = Repository(args.output_dir, clone_from=repo_name, token=args.hub_token)
+
+        elif args.output_dir is not None:
+            os.makedirs(args.output_dir, exist_ok=True)
+
+    # For testing use a dummy model
+    if args.testing: 
+        config = UNet2DModel.load_config('diffusers/consistency-models-test', subfolder="test_unet")
+    elif args.model_config_name_or_path is not None:
+        config = UNet2DModel.load_config(args.model_config_name_or_path)
+    # Use the config if provided, model and target model have the same structure
+    if config is not None:
+        model = UNet2DModel.from_config(config)
+        target_model = UNet2DModel.from_config(config)
+    # Otherwise, use a default config
+    else:
+        model = UNet2DModel(
+                sample_size= args.resolution,
+                in_channels=3,
+                out_channels=3,
+                layers_per_block=2,
+                num_class_embeds=1000,
+                block_out_channels= [32, 64],
+                attention_head_dim=8,
+                down_block_types= [
+                    "ResnetDownsampleBlock2D",
+                    "AttnDownBlock2D",
+                ],
+                up_block_types= [
+                    "AttnUpBlock2D",
+                    "ResnetUpsampleBlock2D",
+                ],
+                resnet_time_scale_shift="scale_shift",
+                upsample_type="resnet",
+                downsample_type="resnet"
+                )
+        target_model = UNet2DModel(
+                sample_size= args.resolution,
+                in_channels=3,
+                out_channels=3,
+                layers_per_block=2,
+                num_class_embeds=1000,
+                block_out_channels= [32, 64],
+                attention_head_dim=8,
+                down_block_types= [
+                    "ResnetDownsampleBlock2D",
+                    "AttnDownBlock2D",
+                ],
+                up_block_types= [
+                    "AttnUpBlock2D",
+                    "ResnetUpsampleBlock2D",
+                ],
+                resnet_time_scale_shift="scale_shift",
+                upsample_type="resnet",
+                downsample_type="resnet"
+                )    
```

Comment:
could we remove this? would prefer to not hardcode a config in the training script